test/src/test/java/AbstractFeatureServiceTest.java

```diff
@@ -28,17 +28,6 @@ public void setupRestAssured() {
             host = "localhost";
         }
         RestAssured.baseURI = "http://" + host;
-
-        String user = System.getProperty("featureServer.user");
```

Comment:
This wasn't having any impact, since the default Koop server requires no authentication. This will change a bit in DEVEXP-222, when we need to configure RestAssured based on the auth type of the Koop server that a particular test is hitting.